cli/src/cli/run.rs

```diff
@@ -32,6 +32,11 @@ pub fn run(
         .load_config()
         .map_err(|err| format_err!("{}", err))?;
 
+    println!(
```

Comment:
Won't this be a weird message if p2p_bindings is None?

Reply:
Yeah, kind of. Us lazy devs are used to weird messages like that :)

Reply:
Could you make it conditional?

Reply:
Us obsessive documenters go nuts over messages like that :p

Reply:
You can use [ci skip] and [skip travis] in the commit message to prevent rebuilding

Reply:
i'll just erase it